README.md

```diff
@@ -2,20 +2,20 @@
 
 # Jaeger Bindings for PHP OpenTracing API
 
-This is a client-side library that can be used to instrument PHP apps for distributed trace collection, and to send those traces to Jaeger. See the [OpenTracing PHP API](https://github.com/opentracing/opentracing-php) for additional detail.
+This is a client-side library that can be used to instrument PHP apps for distributed trace collection,
+and to send those traces to Jaeger. See the [OpenTracing PHP API](https://github.com/opentracing/opentracing-php)
+for additional detail.
 
 ## Contributing and Developing
 
 Please see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 ## Installation
 
-```json
-{
-  "require": {
-    "jonahgeorge/jaeger-client-php": "0.3.0"
-  }
-}
+Install [Composer](https://getcomposer.org) in a common location or in your project. Then run the installer as follows:
+
+```bash
+php composer.phar require jonahgeorge/jaeger-client-php
```

Comment:
Really just the smallest nit: Can we get rid of the composer install instruction and do
```sh
composer require <>
```
instead of the phar

Reply:
Fixed! :)